### PR TITLE
10 new companies/projects that use Dai

### PR DESCRIPTION
**Pull request by acolytec3**
Added 10 new places that accept Dai and a new category (Payments). 

New projects: OpenSea, Aave Pay, Dai Card, Coingate, Wirex, Nuo, TokenSets, AirSwap, Bounties Network, Bounty0x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,23 +48,38 @@
...
+* [Nuo](https://www.nuo.network/) - Lend, borrow, and margin trade your Dai, ETH, and other ERC20 tokens
+* [TokenSets](https://www.tokensets.com/) - Automated rebalancing of crypto portfolios supporting Dai, ETH, and other tokens
 * [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
 
-
 ## Freelancers
 
+* [Bounties Network](https://bounties.network/) - Earn Dai, ETH, or other tokens by working on freelance projects or fund your own
+* [Gitcoin](https://gitcoin.co/) - Platform for crowdfunding and freelance developers for your software projects.
+* [Bounty0x](https://https://bounty0x.io/) - Fund or complete bounties for  marketing, development, and creative tasks with/for Dai, ETH, and other tokens.
```
Comment (apbendi) [inline README.md]: Link is broken here (2x https)

File: README.md [hunk 2]
```diff
@@ -47,23 +48,38 @@
...
 
 * [Bidali](https://giftcards.bidali.com/buy-giftcards-with-dai/) - Buy gift cards with Dai.
+* [OpenSea](https://opensea.io) - Buy and sell digital good and assets.  Make offers in Dai.
+
+## Payments
+
+* [Aave Pay](https://pay.aave.com/) - Use Dai and other crypto to send fiat payments to basically anyone.
+* [Coingate](https://coingate.com) - Accept payment in Dai and 50 other crypto currencies.
+* [Dai Card](https://daicard.io/) - Instant, off-chain, peer to peer, Dai transactions using browser-based UI.
+* [Wirex](https://wirexapp.com/card/) - Fund a Wirex debit card with Dai and spend anywhere Visa is accepted.
```
Comment (apbendi) [inline README.md:62]: Couldn't find any mention of Dai on their site. Where did it say they allow you to use it?

**Review by apbendi — CHANGES_REQUESTED** [review]
Great work on this! I've added a few minor points of feedback, but it looks great overall 🎉 Thanks for getting it done in a timely manner.

File: README.md (hunk 2 above)
Comment (acolytec3) [reply README.md:62]: https://wirexapp.com/blog/post/a-token-to-dai-for-0004 According to this blog entry, they do support it. I found mentions on Reddit of people using Dai with a wirex cars too.

File: README.md (hunk 1 above)
Comment (acolytec3) [reply README.md]: Fixed

**Pull request merged**